### DISCUSSION TITLE
Finetune the new pretrained French VITS model

### DISCUSSION THREAD

**Discussion by JeanLorys** [coqui-ai/TTS#2173] [General Q&A]
Hello,

I saw that a new VITS model pre-trained in French was released with Coqui.ai v.0.9.0. 
However, we only have access to a `model_file.pth.tar` and not a `model_file.pth`. Therefore, I can't restore the model correctly as I could do with the French pre-trained Tacotron-2 model because the `tar` extension is not supported by the `formatter.py`.

Here is the error :
`ValueError : Unsupported file type`.

Could you help me to solve this problem?

Thanks in advance!

**Comment by erogol** [comment]
Try renaming the file manually.
